openy.profile

```diff
@@ -229,6 +250,58 @@ function openy_install_features(array &$install_state) {
   return ['operations' => $module_operations];
 }
 
+/**
+ * Create batch for installing search.
+ *
+ * @param array $install_state
+ *   Installation parameters.
+ *
+ * @return array
+ *   Batch.
+ */
+function openy_install_search(array &$install_state) {
```

Comment:
Please define the variable with service and use it everywhere in the function.

Reply:
@anprok, this one is not fixed, I am talking about `\Drupal::state()` here. Please define a variable with the service with PHP doc and use it everywhere in the function.